### PR TITLE
Create zva_wmk_de.py

### PR DESCRIPTION
**Pull request by Demel75**
Added source and documentation for Werra-Meißner-Kreis, Hesse, germany

Hint: URL for 2022 is different to the from 2019..2021 and 2023.

### CODE REVIEW

**Review by mampfes — CHANGES_REQUESTED** [review]
Please add new source also to info.md and README.md

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if year == 2022:
+           yearstr = ""
+           street = self._street.upper()
+        else:
+           yearstr = ("-" + str(year))
+
+        urlzva = "https://www.zva-wmk.de/termine/schnellsuche"+yearstr+"&type=all&link=ical&timestart=6&fullday=1&timeend=17&reminder=1440&display=0&city="+city+"&street="+street
+
+        r = requests.get(urlzva)
+        r.encoding = r.apparent_encoding
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:44]: Don't use `apparent_encoding `but specify the correct encoding (typically `utf-8`) directly.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        self._city = city
+        self._street = street
+        self._ics = ICS(split_at=" / ")
+
+    def fetch(self):
+        city = self._city.upper()
+        city = self._city.replace('ß', 'ẞ').upper()
+        city = city.replace(" - ", "_")
+        city = city.replace(" ", "+")
+        city = city.replace("ẞ", "ß")
```
[the remark this answers is not in the document]
Comment (Demel75) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:29]: yes, it's a upper ß and a lower ß. Upper by Python makes ß --> SS

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py (hunk 1 above)
Comment (Demel75) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/zva_wmk_de.py:44]: removed

**Pull request closed without merging**